pontoon/contributors/views.py

```diff
@@ -471,6 +471,29 @@ def mark_all_notifications_as_read(request):
     return JsonResponse({"status": True})
 
 
+@login_required(redirect_field_name="", login_url="/403")
+@require_POST
+@transaction.atomic
+def toggle_user_status(request, username):
+    # only admins are authorized to (dis|en)able users
+    if not (
+        request.user.is_authenticated
+        and (request.user.is_staff or request.user.is_superuser)
```

Comment:
The decorator already checks that the user is authenenticated. 

And same as in the template, you can drop the `is_staff` check here, too.